### ISSUE TITLE
Incorrect total on Ubuntu 20.04 (Shows 20.4 GiB used and free shows 9.7GiB used)

### ISSUE THREAD

**Issue by shanness** [pixelb/ps_mem#68]
```
craig@home:~ (master)$ ps_mem | head -2
 Private  +   Shared  =  RAM used	Program

craig@home:~ (master)$ ps_mem | tail -3
---------------------------------
                         20.2 GiB
=================================
craig@home:~ (master)$ free -h
              total        used        free      shared  buff/cache   available
Mem:           62Gi       9.5Gi        12Gi       419Mi        40Gi        52Gi
Swap:            0B          0B          0B
craig@home:~ (master)$ uname -a
Linux home 5.9.16-050916-generic #202012211331 SMP Mon Dec 21 14:11:13 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux

craig@home:~ (master)$ ps_mem --version
3.12
```

Also tested with 3.13 (downloaded, but not showing in pip) with same results.

**Comment by mraof** [comment]
This seems to have been fixed with https://github.com/pixelb/ps_mem/commit/0daa345f22ebce0a5e7f0513f9172ece7227d70b but the released 3.13 version predates that fix

**Comment by shanness** [comment]
3.14 is now out :+1: and it's fixed in it, thanks!
